### DISCUSSION TITLE
ERROR: Token is issued in the future (iat). when running prisma deploy for production environment

### DISCUSSION THREAD

**Discussion by ralucafleseriu** [prisma/prisma#2151] [Q&A] [answered]
I get this error when trying to run prisma deploy , form local env to production env.
```
Creating stage default for service default !
ERROR: Token is issued in the future (iat).

{
  "data": {
    "addProject": null
  },
  "errors": [
    {
      "locations": [
        {
          "line": 2,
          "column": 9
        }
      ],
      "path": [
        "addProject"
      ],
      "code": 3015,
      "message": "Token is issued in the future (iat).",
      "requestId": "local:ck8t82idw1fl90725fogs8xdy"
    }
  ],
  "status": 200
}

```
Any idea why and how to debug this? It worked fine until last deploy

**Comment by janpio** [comment, marked as answer]
That sounds like a Prisma 1 problem, you probably want to ask this at https://github.com/prisma/prisma1 🙇

**Reply by ralucafleseriu** [reply]
Thanks a lot! 
I couldn't understand why all my google search results were giving me 404 pages... :))  
This 404 page in google results might be an issue for other people too.. not sure how it should be fixed .

Fixed the problem by changing my local machine's time

**Reply by Sytten** [reply]
Yeah the rename of prisma2 to prisma (and prisma to prisma1) is causing a lot of confusion in the community.

**Reply by ralucafleseriu** [reply]
Yap.. more because the prisma 2 is a beta version, right? Didn't expect having it as the main prisma

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
